Sources/KeyboardShortcuts/util.swift

```diff
@@ -3,7 +3,7 @@ import Carbon.HIToolbox
 
 
 extension String {
-	/// localization
+	/// Localize a string
```

Comment:
It's not "a" string, it's "the" string or "this" string.

A better description would be:

> Makes the string localizable.